Formula/webdis.rb

```diff
@@ -0,0 +1,37 @@
+class Webdis < Formula
+  desc "Redis HTTP interface with JSON output"
+  homepage "http://webd.is/"
+  url "https://github.com/nicolasff/webdis/archive/0.1.2.tar.gz"
+  sha256 "8e46093af006e35354f6b3d58a70e3825cd0c074893be318f1858eddbe1cda86"
+
+  depends_on "libevent"
+
+  def install
+    system "make"
+    bin.install "webdis"
+
+    inreplace "webdis.prod.json" do |s|
+      s.gsub! "/var/log/webdis.log", "#{var}/log/webdis.log"
```

Comment:
Thanks for pointing that out @DomT4.

@freeatnet This "roaming neckbeard" works in their free time to make software that you get to use for free. This "roaming neckbeard" decided that, because it wasn't documented, they'd make the change for you locally which is not what we normally do for open-source contributors. This "roaming neckbeard" is having a pretty stressful life at the moment and comments like this make it far worse. Thanks for the kind words.